Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingTreeMac.mm

```diff
@@ -579,6 +579,12 @@ static bool layerEventRegionContainsPoint(CALayer *layer, CGPoint localPoint)
     });
 }
 
+void RemoteScrollingTreeMac::CAFrameLogging(CFTimeInterval beginTime, CFTimeInterval commitDeadline)
```

Comment:
This method name is not great (if it's even necessary once you move this into a wrapper object).

Reply:
(if you don't factor it out, which I hope you do, I can try to help come up with a better name)